src/Robot.cpp

```diff
@@ -67,8 +70,8 @@ void Robot::TeleopInit() {
 	if (autonomousCommand.get() != nullptr)
 		autonomousCommand->Cancel();
 	//Setting starting values of Pneumatics......................
-	RobotMap::chassisshiftLeft->Set(DoubleSolenoid::kReverse);
-	RobotMap::chassisshiftRight->Set(DoubleSolenoid::kReverse);
+	RobotMap::chassisshift->Set(DoubleSolenoid::kReverse);
```

Comment:
This means we are tieing both shifter on the same line correct? I originally thought it would be like this. Just make sure to explain to marty why (he originally wored on the shifters)